users/urls.py

```diff
@@ -52,5 +53,10 @@
         ),
         name="users_api-current_user",
     ),
+    path(
+        "api/userinfo",
```

Comment:
Could you have this be api/v0 because that's where we namespace internal APIs?
```suggestion
        "api/v0/userinfo",
```